mlos_core/mlos_core/optimizers/optimizer.py

```diff
@@ -84,16 +95,16 @@ def register(
 
         Parameters
         ----------
-        configs : pd.DataFrame
+        configs : pandas.DataFrame
```

Comment:
should we `import pandas` instead of `import pandas as pd` like we do now to be consistent?

Reply:
Probably could, but I think this made it easier for the docs parser to find the relevant references, so I'd rather leave it as is since I know it's working :)